contracts/Tokenomics.sol

```diff
@@ -962,13 +950,6 @@ contract Tokenomics is TokenomicsConstants {
                     if (!mapNewUnits[unitType][serviceUnitIds[j]]) {
                         mapNewUnits[unitType][serviceUnitIds[j]] = true;
                         mapEpochTokenomics[curEpoch].unitPoints[unitType].numNewUnits++;
-                        // Check if the owner has introduced component / agent for the first time
-                        // This is done together with the new unit check, otherwise it could be just a new unit owner
-                        address unitOwner = IToken(registries[unitType]).ownerOf(serviceUnitIds[j]);
-                        if (!mapNewOwners[unitOwner]) {
-                            mapNewOwners[unitOwner] = true;
-                            mapEpochTokenomics[curEpoch].epochPoint.numNewOwners++;
-                        }
```

Comment:
why removed?

Reply:
This is no longer used anywhere in the calculations.